src/frontend/src/pages/CreateProjectPage/CreateProjectForm.tsx

```diff
@@ -38,13 +40,9 @@ const CreateProjectForm: React.FC = () => {
 
   const handleCancel = () => history.goBack();
 
-  const redirectToCrTable = () => history.push(routes.CHANGE_REQUESTS);
-
   const handleSubmit = async (e: any) => {
```

Comment:
Should this be left as `any`?

Reply:
also will get changed whenever we move to react hook forms